src/tasks/DeliveryTask.py

```
@@ -419,8 +419,8 @@ def other_run(self):
         self.try_time = 0
         self.ensure_main(time_out=120)
         self.log_info("前置操作：按Y，点击‘仓储节点’，点击‘运送委托列表’")
```

Ruff (0.15.14)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)